.github/workflows/swift-ci.yml

```
@@ -0,0 +1,91 @@
+#
+# .github/workflows/swift-ci.yml
+#
+---
+name: Swift CI
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build-swift:
+    name: Build Swift (macos-latest)
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Select Xcode
+        run: |
+          # Use the latest stable Xcode version
+          sudo xcode-select -s /Applications/Xcode.app/Contents/Developer
+          xcodebuild -version
+
+      - name: Create .env file
+        run: |
+          echo "DITTO_APP_ID=test_app_id" > .env
+          echo "DITTO_PLAYGROUND_TOKEN=test_playground_token" >> .env
+          echo "DITTO_AUTH_URL=https://auth.example.com" >> .env
+          echo "DITTO_WEBSOCKET_URL=wss://websocket.example.com" >> .env
+
+      - name: Generate Env.swift
+        working-directory: swift
+        run: |
+          chmod +x buildEnv.sh
+          ./buildEnv.sh ../.env tasks/
+
+      - name: Resolve Swift Package Dependencies
+        working-directory: swift
+        run: |
+          xcodebuild -resolvePackageDependencies \
+            -project Tasks.xcodeproj \
+            -scheme Tasks
+
+      - name: Build Swift Project
+        working-directory: swift
+        run: |
+          # First, let's check available platforms
+          xcodebuild -showsdks || true
+          
+          # Build for generic iOS device (not simulator-specific)
```

The comment is contradicting the actual `-destination`.

```suggestion
          # Build for iOS simulator
```